### ISSUE TITLE
check_whitespace/arrow_alignment doesn't handle all cases properly

### ISSUE THREAD

**Issue by gerases** [puppetlabs/puppet-lint#174]
## Describe the Bug
Given code similar to below:

```
someresource { 'title':
  foo          => 'bar',
}
```

No warnings are raised

## Expected Behavior

I would expect the extra white space to be flagged, but I realize this might have been outside the scope of this plugin. If this is the case, can you recommend the plugin needed?

**Comment by gerases** [comment]
Upon further contemplation, I realized that it's probably outside the scope of this arrow_alignment plugin. So I wrote a separate plugin to handle this case. I will submit a PR within the coming days.

**Comment by kenyon** [comment]
The separate PR was #175.
